### ISSUE TITLE
savePrivateTabsInSessions doesn't work

### ISSUE THREAD

**Issue by hidavy** [Infocatcher/Private_Tab#194]

**Comment by Infocatcher** [comment]
See #82. Now Firefox itself don't save private tabs.
So, now all filtrations are hardcoded inside built-in sessions mechanism.

Also there is AMO policies that prohibit such things:
https://blog.mozilla.org/addons/2010/02/23/private-browsing-support-required-for-add-ons/

**Comment by hidavy** [comment]
OK. I wish there could be a way to not save history for certain tabs yet still kept in sessions. This function would be useful.

**Comment by Infocatcher** [comment]
Yes, I understand. And that's why there was `savePrivateTabsInSessions` preference (for debug purposes, but not only).
But for now it's very difficult to alter built-in things (hard to do it correct and very hard to make something robust).
